src/application.c

```diff
@@ -282,6 +282,43 @@ void save_clicked_handler(GtkWidget *widget, struct swappy_state *state) {
   save_state_to_file_or_folder(state, NULL);
 }
 
+void save_as_clicked_handler(GtkWidget *widget, struct swappy_state *state) {
+  gchar *filename_suggestion;
+  GtkWidget *dialog;
+  GtkFileChooser *chooser;
+  GtkFileChooserAction action = GTK_FILE_CHOOSER_ACTION_SAVE;
+  gint res;
+
+  commit_state(state);
+  filename_suggestion = format_filename(state->config->save_filename_format);
+
+  dialog = gtk_file_chooser_dialog_new("Save Snapshot",
```

Comment:
Is this displayed anywhere? Because if it is, it should be translatable as well using `gettext`.

Reply:
Hm, that's an excellent question. I don't think so. It's not visible on my setup (archlinux + sway) and I installed Xubuntu 22.04 in a vm, compiled swappy there and checked if I can find it there. I didn't. So, I'm inclined to set it to `NULL` as it would be allowed [according to the documentation](https://docs.gtk.org/gtk3/ctor.FileChooserDialog.new.html#parameters).

Reply:
I think it's fine to set `NULL` for now. I'll do some more testing on gnome and sway.